Sources/swift-build/usage.swift

```diff
@@ -26,6 +26,8 @@ func usage(_ print: (String) -> Void = { print($0) }) {
     print("  --fetch                        Fetch package dependencies")
     print("  --update                       Update package dependencies")
     print("  --generate-xcodeproj[=<path>]  Generates an Xcode project [-X]")
+    print("  --show-dependencies[=<mode>]   Print dependency graph (text|dot|json)")
+    print("  --show-package                 Print the contents of Package.swift")
```

Comment:
I would be a little inclined to call this `--dump-package` to differentiate it from something a typical developer would use commonly.

I might also consider dropping it into a different section in the `--help`.